index.js

```
@@ -33,19 +41,49 @@ var loadServerFunctions = function() {
+        })
+    }
+
+    try {
+        const sfmodules = fs.readdirSync("./src/modules")
+        sfmodules.forEach(item => {
+            globfolder = item
+            const innerfiles = fs.readdirSync("./src/modules/" + globfolder + "/backend")
+            //console.log(innerfiles)
+            innerfiles.forEach(item => {
+                code = fs.readFileSync("./src/modules/" + globfolder + "/backend/" + item, 'utf8');
+                eval(code);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)